src/hdf5plugin/_version.py

```diff
@@ -42,6 +42,8 @@ class _VersionInfo(NamedTuple):
     def from_string(cls, version: str) -> "_VersionInfo":
         pattern = r"(?P<major>\d+)\.(?P<minor>\d+)\.(?P<micro>\d+)((?P<prerelease>a|b|rc)(?P<serial>\d+))?"
         match = re.fullmatch(pattern, version, re.ASCII)
+        if match is None:
+            raise RuntimeError(f"Cannot parse version: {version}")
```

Comment:
Spotted by mypy: make sure regexp is matched

Reply:
indeed :+1: